.github/CODEOWNERS

```
@@ -1,4 +1,4 @@
-* @primer/rails-reviewers
+* @primer/rails-reviewers @primer/engineer-reviewers
```

```suggestion
* @primer/engineer-reviewers
```

Already created the team so we can start using it